LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
FYI @nate-ni I made a few changes that I pushed directly to your branch.
- I generated the LICENSE file from GitHub's new repo template. This updated the name to `LICENSE` (no extension) and adjusted the whitespace. I think that is preferable to help automated tooling identify the file more consistently
- Because package.json files were updated I ran `npm install` in the root of the project and checked in the resulting `package-lock.json` file changes. Anytime a `package.json` file is edited you need to do that workflow
- I updated the blazor files to use [PackageExpressions](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata) as we now have a standard SPDX license instead of a custom one.

Reply:
Thanks for doing that.